src/App.tsx

```diff
@@ -1,7 +1,35 @@
-import React from 'react';
+import { Component } from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  lastPressedKey: string;
+};
+
+export class App extends Component<{}, State> {
+  state = {
+    lastPressedKey: 'Nothing was pressed yet',
```

Comment:
'Nothing was pressed yet' - is not a key, let's keep single responsibility pattern, so lastPressedKey is used only for saving a key which was pressed

Reply:
set it '' by default and  just check in your JSX is some key pressed if no - show the default message